tools/update-pdfjs

```diff
@@ -17,6 +17,14 @@ DEST_DIR=src/vendor/pdfjs
 PREFIX=pdf.js-gh-pages
 COMPONENTS="$PREFIX/build $PREFIX/web $PREFIX/LICENSE"
 
+# Check for uncommitted git changes. See https://stackoverflow.com/a/3879077/434243
+git update-index --refresh
+git diff-index --quiet HEAD --
+if [[ $? -ne 0 ]]; then
+  echo "Cannot update PDF.js when there are uncommitted changes in working tree."
+  exit 1
+fi
+
```

Comment:
Gah. I hadn't actually intended to commit this in this PR, although it would be useful to have. This is an example of why it would be good to avoid duplicating this script across repositories: https://github.com/hypothesis/client/issues/4397.

Reply:
Heh. "The safety net is working."